modules/core/core-api/src/main/java/com/enonic/xp/sortvalues/SortValuesProperty.java

```diff
@@ -0,0 +1,47 @@
+package com.enonic.xp.sortvalues;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class SortValuesProperty
```

Comment:
PublicApi

Reply:
This class participated in equals hashcode but doesn't define own. I don't like them here (probably remove them in xp8) but for now for consistency they should be defined